setup.sh

```diff
@@ -26,6 +26,8 @@ echo
 read -p "Browser to use (default: /Applications/Safari.app/) > "  BROWSER
 BROWSER=${BROWSER:-"/Applications/Safari.app/"}
 
+/bin/bash
```

Comment:
Is this supposed to be here?

Reply:
Hmm no I don't think so, let me get rid of it.